dist/api_data.py

```diff
@@ -743,6 +743,10 @@ def __ge__(self, other):
             number of bytes per second available to the chunk cache. The minimum non-zero setting
             is 1MB.''',
             min='0', max='1TB'),
+        Config('fsync_background_period_sec', '0', r'''
+            max waiting period for background fsync. If the written threshold is not met in this time,
+            a background fsync is done.''',
+            min='0', max='60'),
```

Comment:
Can you change the stat name to `fsync_maximum_wait_period` please, both here and in other parts of the code? I think that would be a closer match to the behaviour.